spec/olm/install/install_test.go

```diff
@@ -0,0 +1,44 @@
+package install
+
+import (
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo"
+	"github.com/openshift-pipelines/release-tests/pkg/operator"
+)
+
+var Given = ginkgo.Describe
+var Then = ginkgo.It
```

Comment:
1. I'm used to Describe & It so for me personally this is unnecessary
2. When we want to use Given, When & Then, then let's define it globally, not in every test suite.

Reply:
Globally Doesn't make sense, As we have multiple components , It's better Use Describe and It should makes sense, updated code base